views/layout.jsx

```diff
@@ -2,24 +2,74 @@ import { createElement } from "complate-stream";
 
 let CSS = `
```

Comment:
Quick note because I ran into this the other day: `<style>{CSS}</style>` should be changed to `<style>{safe(CSS)}</style>`, otherwise `>` selectors will be escaped.

Reply:
Just updated it.

Reply:
Although honestly I probably wouldn't have a problem with putting the CSS in a file and loading it instead of writing it inside a JavaScript variable.

Reply:
Heh totally agree; this is probably a remnant of the original "done is better than perfect" approach. But let's not get bogged down by technology and do the Simplest Thing™ for now: I like that you've focused on content so far, we can always extract CSS later on.

Reply:
Agreed. :)